test/plots/empty.js

```diff
@@ -11,6 +11,8 @@ export default async function () {
       Plot.frame(),
       undefined,
       null,
+      false,
+      "",
```

Comment:
```suggestion
      false,
      "",
      0,
```

Reply:
I can see how respecting _false_ could be useful; not sure about considering `""` and `0` (Plot tends to be very deliberately "strict").